### PR TITLE
Fixed sales item suggestion by location

### PR DESCRIPTION
**Pull request by ri-cky**

### CODE REVIEW

File: .github/workflows/new-build-image.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Image version"
+        required: true
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
```
Comment (jekkos) [inline .github/workflows/new-build-image.yml:12]: Cool you are using github actions here to build the project? #2834

File: application/models/Item.php [hunk 2]
```diff
@@ -544,17 +544,32 @@ private function append_label(&$label, $item_field_name, $item_info)
 		}
 	}
 
-	public function get_search_suggestions($search, $filters = array('is_deleted' => FALSE, 'search_custom' => FALSE), $unique = FALSE, $limit = 25)
+	public function get_search_suggestions($search, $filters = array('is_deleted' => FALSE, 'search_custom' => FALSE, 'location' => NULL), $unique = FALSE, $limit = 25)
```
Comment (jekkos) [inline application/models/Item.php:547]: this is probably the change you wanted to submit? but the changeset seems to be made against old 3.3.9. Currently we only accept contributions against master branch which has moved to CI4.

**Review by jekkos — CHANGES_REQUESTED** [review]

File: application/models/Item.php (hunk 2 above)
Comment (jekkos) [reply application/models/Item.php:547]: should not be so hard to port it to the master branch, if you feel up to that, then that would be great!

**Comment by ri-cky** [comment]
@MalikAbiola Please check the comment and conflict in this PR

File: application/models/Item.php (hunk 2 above)
Comment (ri-cky) [reply application/models/Item.php:547]: I will try to do that and make a fresh PR

**Pull request closed without merging**